### PR TITLE
[System Requirements] Check for ImageMagick delegate LCMS

### PR DESCRIPTION
**Pull request by BlackbitDevs**
As already described in #6790 there can be a problem with negative colors when generating thumbnails from CMYK images. As proposed in https://talk.pimcore.org/t/cmyk-to-rgb-displaying-as-negative/908/17 and https://www.php.net/manual/en/imagick.profileimage.php#114781 this is caused by ImageMagick missing the `lcms` delegate. This PR adds a check in the system requirements for this delegate.

### CODE REVIEW

File: lib/Tool/Requirements.php [hunk 1]
```diff
@@ -615,6 +616,16 @@ public static function checkPhp()
             'state' => class_exists('Imagick') ? Check::STATE_OK : Check::STATE_WARNING,
         ]);
 
+        if(class_exists('Imagick')) {
+            $imageMagickLcmsDelegateInstalledProcess = Process::fromShellCommandline('convert -list configure | grep DELEGATES');
```
Comment (brusch) [inline lib/Tool/Requirements.php]: We should use `\Pimcore\Tool\Console::getExecutable()` to determine the path to `convert`.

File: lib/Tool/Requirements.php [hunk 2]
```diff
@@ -615,6 +616,16 @@ public static function checkPhp()
...
+            $imageMagickLcmsDelegateInstalledProcess->run();
+            $checks[] = new Check([
+                'name' => 'ImageMagick LCMS delegate',
+                'link' => 'https://talk.pimcore.org/t/cmyk-to-rgb-displaying-as-negative/908/17',
```
Comment (brusch) [inline lib/Tool/Requirements.php]: I'd add a note about LCMS here: https://github.com/pimcore/pimcore/blob/7fff27a970182dc18717b3aa1727d29e388ba72d/doc/Development_Documentation/23_Installation_and_Upgrade/01_System_Requirements.md#L40 and change the link to the docs.

File: lib/Tool/Requirements.php (hunk 1 above)
Comment (brusch) [reply lib/Tool/Requirements.php]: I'd prefer to not use `grep` but pure PHP.

**Review by brusch — CHANGES_REQUESTED** [review]

File: lib/Tool/Requirements.php (hunk 1 above)
Comment (BlackbitDevs) [reply lib/Tool/Requirements.php]: Done.

File: lib/Tool/Requirements.php (hunk 2 above)
Comment (BlackbitDevs) [reply lib/Tool/Requirements.php]: Done.

**Comment by brusch** [comment]
Very nice, thanks a lot! 👍

**Pull request merged**